README.md

```diff
@@ -1,2 +1,16 @@
 # Hello-Microverse
-This is my first project at Microverse for mastering the basics of repo
+This is my first project at Microverse for mastering the basics of setting up repo 
+
+## Built with
+
+* html
+* css
+* linter
+
+## Getting started 
+this example shows you how to set up a repository in a professional manner 
+and it will be used through out all the Microverse program .
+
+## Authors
+* GitHub: [yasinabdmahmood](https://github.com/yasinabdmahmood)
+* Twitter: [@YASEEN30093859](https://twitter.com/yasenabd7)
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly update and add the following missing headers from the [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md)

1. Getting started: In this header, you should add some instructions on how a user can have your repo locally on their machine. (hot to clone the repo, how to move into the cloned repo), 
2. Prerequisite: e.g `vscode editor`, a `web browser(chrome, etc)`.
3. Contribution header.
4. Show your support header.
5. Acknowledgments.
6. License.
:point_right: The arrangement should follow that use in the [microverse readme tamplate](https://github.com/microverseinc/readme-template/blob/master/README.md) to keep consistency to make your repo appear professional.

*LINTER*
- As it's a requirement for all projects in microverse, please kindly sure your repo has no linter error.